packages/main/pyproject.toml

```diff
@@ -2,9 +2,7 @@
 name = "rpaframework"
 version = "13.1.0"
```

Comment:
Maybe we should increase library version now or in another subsequent PR once we decide to release this.